chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/developer/attestation/DeveloperAttestationCheckInReportJob.java

```diff
@@ -68,12 +64,12 @@ public void execute(JobExecutionContext context) throws JobExecutionException {
             @Override
             protected void doInTransactionWithoutResult(TransactionStatus status) {
                 try {
-                    List<DeveloperAttestationCheckInReport> reportRows = developerAttestationCheckInReportDataCollection.collect();
-                    File csv = developerAttestationCheckInReportCsvWriter.generateFile(reportRows);
+                    //List<DeveloperAttestationCheckInReport> reportRows = developerAttestationCheckInReportDataCollection.collect();
+                    //File csv = developerAttestationCheckInReportCsvWriter.generateFile(reportRows);
                     chplEmailFactory.emailBuilder()
                             .recipient(context.getMergedJobDataMap().getString("email"))
                             .subject(emailSubject)
-                            .fileAttachments(Arrays.asList(csv))
+                            //.fileAttachments(Arrays.asList(csv))
```

Comment:
Is there a `TODO` associated with this? If we are planning on putting it back in, can we note that with the ticket we're planning on using for it, or remove it otherwise?